dominion/dominion_ui/src/components/HowItWorks/FirstRow.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import HowItWorksContent from './HowItWorksContent';
+
+function FirstRow() {
+  return (
+    <HowItWorksContent
+      title="01. Find Facts"
+      description="Population and financial figures are broken down by category: Elections,
+    Demographics, Service Delivery, Economics, and Education."
```

Comment:
Maybe use template literals.

Reply:
@karimkawambwa I think it works well either way. Feel free to change if you would like to

Reply:
@karimkawambwa The mobile does not necessarily match the desktop design due to the different viewpoint...we do need the text to be mobile friendly and responsive...otherwise the content sections on mobile appear squeezed...